packages/big-design/src/components/Input/Input.tsx

```diff
@@ -3,43 +3,68 @@ import hoistNonReactStatics from 'hoist-non-react-statics';
 import React, { Ref } from 'react';
 
 import { uniqueId } from '../../utils';
+import { Chip } from '../Chip';
 import { Form } from '../Form';
 import { Small } from '../Typography';
 
-import { StyledIconWrapper, StyledInput, StyledInputWrapper } from './styled';
+import { StyledIconWrapper, StyledInput, StyledInputContent, StyledInputWrapper } from './styled';
 
 interface Props {
+  chips?: string[];
   description?: React.ReactChild;
   error?: React.ReactChild | React.ReactChild[];
   iconLeft?: React.ReactChild;
   iconRight?: React.ReactChild;
   label?: React.ReactChild;
   theme?: ThemeInterface;
+  onChipDelete?(chip: string): () => void;
 }
 
 interface PrivateProps {
   forwardedRef: Ref<HTMLInputElement>;
 }
 
+interface InputState {
+  focus: boolean;
+}
+
 export type InputProps = Props & React.InputHTMLAttributes<HTMLInputElement>;
 
-class StyleableInput extends React.PureComponent<InputProps & PrivateProps> {
+class StyleableInput extends React.PureComponent<InputProps & PrivateProps, InputState> {
   static Description = Small;
   static Error = Form.Error;
   static Label = Form.Label;
+
+  readonly state: InputState = {
```

Comment:
🍹I think you might want to add private 🤷‍♀️
```suggestion
  private readonly state: InputState = {
```